test/Exporter/Test/Writer/KnpSnappyWriterTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Test\Source;
+
+use Exporter\Writer\KnpSnappyWriter;
+
+/**
+ * Tests the KnpSnapyWriter Writer.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class KnpSnappyWriterTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var string
+     */
+    protected $filename;
+
+    /**
+     * @var string
+     */
+    protected $wkhtmltopdfBinary;
+
+    public function setUp()
+    {
+        $this->filename = 'foobar.pdf';
```

Comment:
I tried to implement it in the SonataAdminBundle Exporter (`php://output`), works well.